src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -64,31 +64,50 @@
 
 static std::atomic<int64_t> num_dropped_connections{0};
 
+using ::grpc_event_engine::experimental::ConfigMap;
+using ::grpc_event_engine::experimental::EndpointConfig;
+
+namespace {
+ConfigMap CopyFromEndpointConfig(const EndpointConfig& config) {
+  ConfigMap map;
```

Comment:
This could just be a struct of integers and strings

Reply:
Yes thats true but this custom struct would end up getting passed to a lot of places: socket_utils, socket_mutator, grpc_tcp_create (and platform specific implementations of these functions) etc. Rather than being a new struct, I thought maybe it could just be an EndpointConfig type for uniformity

Reply:
I added a new struct now: grpc_tcp_generic_options defined in src/core/lib/iomg/tcp_generic_options{.h,.cc} files